### PR TITLE
UI Enhancements

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
- Fixed Create Shoot Page Layout:
  - Removed `flat` from cards
  - Aligned all component margins
  - Aligned Workers, DNS, Hibernation Management rows
- Use Message slot for Checkbox hint
- Improved Control Plane High Availability Selection
- Moved input classes to main.css
- Removed workaround for vuetify hints (not needed anymore)

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user
NONE
```

### CODE REVIEW

File: frontend/src/components/NewShoot/GNewShootSelectInfrastructure.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <template>
-  <v-row class="my-0">
+  <div class="d-flex">
```
Comment (holgerkoser) [inline frontend/src/components/NewShoot/GNewShootSelectInfrastructure.vue:8]:
```suggestion
  <div class="d-flex justify-start align-center">
```
Does not change the behavior but makes it more clear

File: frontend/src/views/GNewShoot.vue [hunk 2]
```diff
@@ -13,64 +13,53 @@ SPDX-License-Identifier: Apache-2.0
       class="overflow-auto"
       fluid
     >
-      <v-card flat>
+      <v-card>
```
Comment (holgerkoser) [inline frontend/src/views/GNewShoot.vue:16]: Do we also want to change this with this PR?

File: frontend/src/plugins/vuetify.js [hunk 3]
```diff
@@ -68,8 +68,8 @@ const variations = {
     'main-background',
     'toolbar-background',
   ],
-  lighten: 1,
-  darken: 2,
+  lighten: 3,
```
Comment (holgerkoser) [inline frontend/src/plugins/vuetify.js:71]: I am not sure we need this. Let's discuss again?

File: frontend/src/components/ShootHibernation/GHibernationScheduleEvent.vue [hunk 4]
```diff
@@ -289,3 +286,23 @@ export default {
   },
 }
 </script>
+
+<style lang="scss" scoped>
+.regular-input {
+  max-width: 300px;
+  min-width: 180px;
+  flex: 1 1 auto;
```
Comment (holgerkoser) [inline frontend/src/components/ShootHibernation/GHibernationScheduleEvent.vue:294]: Use flex and padding classes from vuetify

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**